src/styles/header.css

```diff
@@ -0,0 +1,6 @@
+.header {
+  margin-top: 40px;
```

Comment:
'header' is your block. We can not give external styling like margin or positioning to block. You can create an element and then give it the external styling.
refer checklist number 6